client/components/backup-retention-management/hooks.tsx

```diff
@@ -1,24 +1,10 @@
-import { useMemo } from '@wordpress/element';
-import { useSelector } from 'react-redux';
-import { useStorageText } from 'calypso/components/backup-storage-space/hooks';
-import getBackupCurrentSiteSize from 'calypso/state/rewind/selectors/get-backup-current-site-size';
-import getRewindBytesAvailable from 'calypso/state/rewind/selectors/get-rewind-bytes-available';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { STORAGE_ESTIMATION_ADDITIONAL_BUFFER } from './constants';
-import type { RetentionRadioOptionType } from './types';
 
 /**
  * Estimate the current site size based on the last backup size, adding a 25% buffer
  */
-export function useEstimatedCurrentSiteSize(): number {
-	const siteId = useSelector( getSelectedSiteId ) as number;
-	const lastBackupSize = useSelector(
-		( state ) => getBackupCurrentSiteSize( state, siteId ) as number
-	);
-	return useMemo(
-		() => lastBackupSize + lastBackupSize * STORAGE_ESTIMATION_ADDITIONAL_BUFFER,
-		[ lastBackupSize ]
-	);
+export function getEstimatedCurrentSiteSize( lastBackupSize: number ): number {
```

Comment:
I think we can remove this function, too. Seems like a straightforward calculation we can do inline.

Reply:
Done!